chromadb/utils/embedding_functions/cloudflare_workers_ai_embedding_function.py

```diff
@@ -0,0 +1,125 @@
+from chromadb.api.types import (
+    Embeddings,
+    Documents,
+    EmbeddingFunction,
+    Space,
+)
+from typing import List, Dict, Any, Optional
+import os
+from chromadb.utils.embedding_functions.schemas import validate_config_schema
+from typing import cast
+
+
+class CloudflareWorkersAIEmbeddingFunction(EmbeddingFunction[Documents]):
+    def __init__(
+        self,
+        model_name: str,
+        account_id: str,
+        api_key: Optional[str] = None,
+        api_key_env_var: str = "CHROMA_CLOUDFLARE_API_KEY",
+    ):
+        """
+        Initialize the CloudflareWorkersAIEmbeddingFunction.
```

Comment:
might be useful to link to CF docs - https://developers.cloudflare.com/workers-ai/models/